Libraries/Lists/VirtualizedList.js

```diff
@@ -1175,9 +1175,12 @@ class VirtualizedList extends React.PureComponent<Props, State> {
               this.context == null
             ) {
               // TODO (T46547044): use React.warn once 16.9 is sync'd: https://github.com/facebook/react/pull/15170
+              // TODO(macOS GH#774): Temporarily ignoring commit 646605b90e666c4b0d1c1200a137eacf62b46f87
```

Comment:
Did you add this TODO or was it already here?

Reply:
I added this TODO.

Reply:
Can you give more context on the errors we're suppressing? I'd default to think we need to update our test app in this case